atom/app/atom_main.cc

```diff
@@ -35,14 +35,14 @@
 #include "atom/common/atom_command_line.h"
 #include "base/at_exit.h"
 #include "base/i18n/icu_util.h"
+#include "electron/buildflags/buildflags.h"
 
 namespace {
 
-#ifdef ENABLE_RUN_AS_NODE
+#if BUILDFLAG(ENABLE_RUN_AS_NODE)
 const char kRunAsNode[] = "ELECTRON_RUN_AS_NODE";
 #endif
 
-#if defined(ENABLE_RUN_AS_NODE) || defined(OS_WIN)
```

Comment:
Why? )

Reply:
The method already has guards wrt implementation, also it seems to be generic, don't see the reason to specifically guard it behind `ELECTRON_RUN_AS_NODE`